test/e2e/far_e2e_test.go

```diff
@@ -346,14 +346,28 @@ func makeNodeUnready(node *corev1.Node) {
 	log.Info("node is unready", "node name", node.GetName())
 }
 
-// checkFarLogs gets the FAR pod and checks whether its logs have logString
-func checkFarLogs(logString string) {
+// buildExpectedLogOutput returns a string with a node identifier and a success message for the reboot action
+func buildExpectedLogOutput(nodeName, successMessage string) string {
+	expectedString := fmt.Sprintf("\"Node name\": \"%s\", \"Response\": \"%s", nodeName, successMessage)
+	log.Info("Substring to search in the logs", "expectedString", expectedString)
+	return expectedString
+}
+
+// checkFarLogs gets the FAR pod and checks whether it's logs have logString
+func checkFarLogs(farNodeName, logString string) {
 	EventuallyWithOffset(1, func() string {
 		pod, err := utils.GetFenceAgentsRemediationPod(k8sClient)
 		if err != nil {
 			log.Error(err, "failed to get FAR pod. Might try again")
 			return ""
 		}
+		if pod.Spec.NodeName == farNodeName {
```

Comment:
Nit: IIUC `farNodeName` is the node that being remidiated.
I think that the name is a bit confusing in that context (initially I thought `farNodeName` is the node that Far Pod is running on).

Reply:
Thanks, it was indeed confusing.